external-crates/move/crates/move-core-types/src/lib.rs

```diff
@@ -28,6 +28,8 @@ pub mod u256;
 mod unit_tests;
 pub mod vm_status;
 
+pub const MAX_VARIANT_COUNT: u64 = 127;
+
```

Comment:
Ah good question. We use this for serializing/deserializing variants. We will refuse to do so if the variant tag is over 127 (to make sure we keep forwards compatibility with uleb128 encoding). 

However, you do make a good point that defining using the value from the file format for this would be better -- will fix.